api/Navigator.json

```
@@ -1156,8 +1156,7 @@
             "chrome": [
               {
                 "version_added": "85",
-                "partial_implementation": true,
-                "notes": "Only supported on Windows. Resolves with an empty array on other platforms."
+                "notes": "Supported on Windows for checking of UWP app installation. Resolves with an empty array on other platforms."
```

Maybe it is worth spelling out UWP?

```suggestion
                "notes": "Supported on Windows for checking of Universal Windows Platform (UWP) app installation. Resolves with an empty array on other platforms."
```